src/shared/components/events-list/index.js

```diff
@@ -57,19 +57,22 @@ export default class EventsList extends Component {
               relevantEvents.map((event) => (
```

Comment:
@lpil's [comment about switching to stateless component](https://github.com/redbadger/website-next/pull/197/files#r70796497) applies here too,